ethers-providers/src/toolbox/pending_transaction.rs

```diff
@@ -56,6 +56,7 @@ use wasm_timer::Delay;
 /// # }
 /// ```
 #[pin_project]
+#[must_use = "A PendingTransaction is a future and it must be polled or awaited to completion"]
```

Comment:
I think this is fitting since sometimes you might forget calling await a second time (`.send_transaction().await.unwrap()` -> `PendingTransaction` -> `.await.unwrap().unwrap()` -> `Receipt`)

Reply:
moved to new branch